server/src/main/java/com/octopus/teamcity/opentelemetry/server/TeamCityBuildListener.java

```diff
@@ -207,12 +207,21 @@ private void createBuildStepSpans(SRunningBuild build, String buildName, Span sp
 
     private void createBlockMessageSpans (List<LogMessage> buildBlockLogs, Span span) {
         for (LogMessage logmessage: buildBlockLogs) {
+            BlockLogMessage parentBlockMessage = logmessage.getParent();
```

Comment:
Consider moving this closer to where it's used

Reply:
done and dusted